lib/components/list/list.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { forwardRef } from "react";
+
+import cn from "classnames";
+
+import type * as Polymorphic from "@radix-ui/react-polymorphic";
+import "./styles.scss";
+
+export interface ListProps {
+  variant?: "base" | "bullets" | "numbers";
+  items: string[];
+}
+
+export type ListComponent = Polymorphic.ForwardRefComponent<"ul", ListProps>;
+
+export const List: ListComponent = forwardRef(function List(
+  { as: ListComp = "ul", variant = "base", items, className, ...props },
+  ref,
+) {
+  return (
+    <ListComp
+      ref={ref}
+      className={cn(
+        "lbh-list",
+        {
+          "lbh-list--bullet": variant === "bullets",
+          "lbh-list--number": variant === "numbers",
+        },
+        className,
+      )}
+      {...props}
+    >
+      {items.map((item, index) => (
+        <li key={index}>{item}</li>
```

Comment:
We might want to just support any react component as an item. I would use the children prop to define the items instead of an explicit prop. Then do something like 
```ts
{React.Children.map(
        children,
        (child) =>
          child &&
          isValidElement(child) &&
          (<li key={key}>{child}</key>)
      )}
```

I've written the above from memory so might not be quite right